Firmware/PlatformIO/main_mcu/src/bms.cpp

```diff
@@ -2,6 +2,14 @@
 
 #include <algorithm>
 
+// Consts for SoE calculation
+const float kDischargeCurrent = 45;
+const float kRegenCurrent = 4.2;
+const float kCellUndervoltage = 2.5;
```

Comment:
```suggestion
const float kCellUndervoltage = 3.23;
```
2.5 would normally be correct, but due to our chargers' limitations, we shouldn't go under 2.83 normally, and we're likely to temporarily be charging half the pack at once, which requires us to stay above 3.23v/cell in order to still be able to charge